db/seeds/utils.js

```diff
@@ -20,3 +22,15 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
+exports.getArticlesByArticleID = (article_id) => {
+  return db 
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then((articles) => {
+      if (articles.rows.length < 1) {
+        return Promise.reject({ status: 404, msg: 'article id does not exist'})
+      } else {
+        return articles.rows
+      }
+    })
+}
```

Comment:
This function looks good in order to check if the article exists later on - however...
I don't think it relates to seeding so it's not stored in the most appropriate place.
Also - if you're bothering to check this - you should have a test in place that gets comments for an article that DOES exist, but has no comments